leap_year.py

```diff
@@ -0,0 +1,34 @@
+class LeapYear:
+    def __init__(self, year_input: int):
+        self._year_input = year_input
+
+    def is_year_input_century_year(self) -> bool:
```

Comment:
Instead of creating 3 methods in the LeapYear class, create a single method as `is_leap_year` and encapsulate the logic inside that. As of now CLI handler class 3 method names. We could have achieved the same behaviour with a single method making the things simple.